.gitignore

```diff
@@ -1,4 +1,5 @@
 _build
 _checkouts
-rebar3.crashdump
+doc
```

Comment:
So I actually want to keep `docs` out of `.gitignore`, and here is why:  We have a `gh-pages` branch, which I am currently using to generate the GitHub pages documentation (https://atomvm.github.io/atomvm_packbeam).  When I generate the docs, I rebase that branch over the tagged release, and I generate the docs on that branch, which get written into the `docs` directory, and I then check in the generated files, which in turn get published to GitHub.  If we git ignored that directory (did you mean `docs` or `doc`?), then I'd end up with a merge conflict on that branch, or I wouldn't be able to check in those files.

So maybe until we have automation enabled, let's live with the inconvenience of these files showing up in `git status`?

Reply:
I did mean `doc`, that is the name of the directory that is created, but I should have realized that this could could interfere with the published documentation. It isn't really an inconvenience, I was more thinking about possible future contributors inadvertently trying to commit the generated directory.